regex/secrets/detected-artifactory-token.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
+- id: detected-artifactory-token
+  pattern-regex: |-
+    AKC[a-zA-Z0-9]{10,}
+  languages: [regex]
```

Comment:
What is the `regex` language?

Reply:
`regex` is an alias for `none`; it means that this rule only uses `pattern-regex` and will operate on text files.